Sources/TripKitUI/helper/Trip+Titles.swift

```diff
@@ -23,16 +23,19 @@ extension Trip {
       departureTimeZone: departureTimeZone,
       arrivalTimeZone: arrivalTimeZone ?? departureTimeZone,
       focusOnDuration: !departureTimeIsFixed,
+      hideExactTimes: hideExactTimes,
       isArriveBefore: request.type == .arriveBefore,
       capitalize: true
     )
   }
   
-  static func timeTitles(departure: Date, arrival: Date, departureTimeZone: TimeZone, arrivalTimeZone: TimeZone, focusOnDuration: Bool, isArriveBefore: Bool, capitalize: Bool = false) -> (title: String, subtitle: String) {
+  static func timeTitles(departure: Date, arrival: Date, departureTimeZone: TimeZone, arrivalTimeZone: TimeZone, focusOnDuration: Bool, hideExactTimes: Bool, isArriveBefore: Bool, capitalize: Bool = false) -> (title: String, subtitle: String) {
     
     let duration = arrival.durationSince(departure)
     
     if focusOnDuration {
+      guard !hideExactTimes else { return (duration, "") }
```

Comment:
This should be done outside the `if`, otherwise a combination of `focusOnDuration == false && hideExactTimes` would still show exact times.

Reply:
Oh okay. I can make this change, though I thought we'd only want to hide this when we want to focus on duration, e.g., in those cases where there's no fixed departure and arrival like PT or frequency-based services?